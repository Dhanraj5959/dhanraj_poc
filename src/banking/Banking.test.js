import { render, screen } from "@testing-library/react";
import { renderComponent } from "../utils/test_helpers";
import BankingStatement from "./Banking";

describe("Home page", () => {
  test("Should Render Intial Page", () => {
    render(renderComponent(<BankingStatement />));
    const text1 = screen.getAllByText(/Banking Statement/i);
  });

  test("Matching the snapshot of the navbar", async () => {
    const { asFragment } = render(renderComponent(<BankingStatement />));
    expect(asFragment()).toMatchSnapshot();
  });
});
