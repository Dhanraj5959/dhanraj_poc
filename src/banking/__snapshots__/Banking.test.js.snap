// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Matching the snapshot of the navbar 1`] = `
<DocumentFragment>
  <div
    class="container mt-5 h-100"
  >
    <div
      class="d-flex justify-content-center h-100"
    >
      <div
        class="card col-5"
      >
        <div
          class="card-header Cardtop "
          style="background-color: black;"
        >
          <h3
            class="login-name text-light"
          >
            <b>
              Banking Statement
            </b>
          </h3>
        </div>
        <h5>
          <p>
            Please withdraw less than 90% of your available balance
          </p>
        </h5>
        <div
          class="card-body"
        >
          <form>
            <div
              class="input-group form-group mt-3"
            >
              <div
                class="input-group-prepend"
              >
                <span
                  class="input-group-text text-white "
                  style="min-width: 150px;"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-unlock fa-beat-fade"
                    data-icon="unlock"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="font-size: 1.75em; color: black;"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M144 144c0-44.2 35.8-80 80-80c31.9 0 59.4 18.6 72.3 45.7c7.6 16 26.7 22.8 42.6 15.2s22.8-26.7 15.2-42.6C331 33.7 281.5 0 224 0C144.5 0 80 64.5 80 144v48H64c-35.3 0-64 28.7-64 64V448c0 35.3 28.7 64 64 64H384c35.3 0 64-28.7 64-64V256c0-35.3-28.7-64-64-64H144V144z"
                      fill="currentColor"
                    />
                  </svg>
                  <label
                    class="text-dark ms-2"
                  >
                    <b>
                      Withdraw
                    </b>
                  </label>
                </span>
              </div>
              <input
                aria-label="number"
                class="form-control ms-2"
                name="number"
                placeholder="Withdraw Amount"
                required=""
                type="number"
                value=""
              />
            </div>
            <div
              class="form-group"
            >
              <button
                aria-label="Login"
                class="btn float-right  btn-dark mt-2 px-4"
                name="Login"
                type="submit"
                value="Login"
              >
                Withdraw
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
